google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -285,6 +285,15 @@ def location
           @gapi.location
         end
 
+        ##
+        # The type of the bucket location.
+        #
+        # @return [String]
+        #
```

Comment:
Please add some documentation here about what this field is and the accepted values

Reply:
Added.